modules/install/pages/thp-disable.adoc

```diff
@@ -237,3 +242,66 @@ If THP is properly disabled, the output of both commands should be the following
 ----
 always madvise [never]
 ----
+
+[#using-thp-service]
+== Using a THP Service
+
+. Create a service file.
++
+[source, console]
+----
+vi /etc/systemd/system/disable-thp.service
+----
+
+. Add the service configuration details to the file and then save it.
++
+[source, console]
+----
+[Unit]
+Description=Disable Transparent Huge Pages (THP)
+DefaultDependencies=no
+After=sysinit.target local-fs.target
+Before=couchbase-server.service
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c 'echo never | tee /sys/kernel/mm/transparent_hugepage/enabled > /dev/null'
+ExecStart=/bin/sh -c 'echo never | tee /sys/kernel/mm/transparent_hugepage/defrag > /dev/null'
+
+[Install]
+WantedBy=basic.target
+----
+
+. Reload the `systemctl` files.
++
+[source, console]
+----
+sudo systemctl daemon-reload
+----
+
+. Ensure that the service will start whenever the system is rebooted.
```

Comment:
Probably best to move this "enable" step to the end, as it was in the DOC ticket.